core-features/studio/test.sh

```diff
@@ -1,13 +1,49 @@
 #!/bin/sh
 
-set -eu
+set -e
```

Comment:
Removed `-u` because we do not want to stop the script for undefined variables (in our case, `PRISMA_FORCE_NAPI` might be undefined)

Reply:
We should use a test here that considers that - most of the other scripts that contain `PRISMA_FORCE_NAPI` do that.